components/brave_sync/time_limited_words_unittest.cc

```diff
@@ -261,4 +261,18 @@ TEST(TimeLimitedWordsTest, GetNotAfterForPureWords) {
   EXPECT_EQ(TimeLimitedWords::GetNotAfter(kValidSyncCode), base::Time());
 }
 
+TEST(TimeLimitedWordsTest, GetWordsCount) {
+  // Normal flow
+  EXPECT_EQ(TimeLimitedWords::GetWordsCount(kValidSyncCode), 24);
+  // Empty case
+  EXPECT_EQ(TimeLimitedWords::GetWordsCount(""), 0);
+  EXPECT_EQ(TimeLimitedWords::GetWordsCount("   "), 0);
```

Comment:
can we add `\t` and `\n` into this case?

Reply:
yes, updated at https://github.com/brave/brave-core/pull/24186/commits/55092e55675edc51f7a950e5070a7e6d3021d418 thanks